4061.json

```diff
@@ -0,0 +1,8 @@
+{
+  "source": "./esy.json",
+  "override": {
+    "resolutions": {
+      "ocaml": "ulrikstrid/ocaml:package.json#443a110eb57d9ca841d4f027a3c6dbf8a12cba45"
```

Comment:
When this get merged we can point to the proper branch: https://github.com/BuckleScript/ocaml/pull/36